target_chains/ton/contracts/tests/PythTest.spec.ts

```diff
@@ -1110,6 +1119,91 @@ describe("PythTest", () => {
     );
   });
 
+  it("should successfully parse price feed updates with more than 3 price feed ids", async () => {
```

Comment:
can you add checks for the last feed in the list as well? just to make sure all work fine.